app/src/components/form/submit.rs

```diff
@@ -0,0 +1,24 @@
+use web_sys::MouseEvent;
+use yew::{function_component, html, Callback, Properties};
+
+#[derive(Properties, PartialEq)]
+pub struct Props {
+    pub disabled: bool,
+    pub onclick: Callback<MouseEvent>,
+}
+
+#[function_component(Submit)]
```

Comment:
Can you add a TODO to migrate the other forms to use this submit (and actually the other reusable components as well)?